### ISSUE TITLE
Building issue on Xcode 12.5

### ISSUE THREAD

**Issue by somenkovnikita** [infobip/mobile-messaging-sdk-ios#32]
Hi,

On Xcode 12.4 we just saw the warnings, but now on the new Xcode 12.5 b3 they have become errors. Xcode 12.5 will be a release 23.03 +- week, I think that this should be taken care of now. Not sure how much I can just remove these attributes.

Error:

```
<prj>/Pods/MobileMessaging/Classes/Vendor/CryptoSwift/Generics.swift:17:28: error: 'exported: true' has no effect in '_specialize' attribute
@_specialize(exported: true, where T == UInt8)
                           ^
<prj>/Pods/MobileMessaging/Classes/Vendor/CryptoSwift/Generics.swift:35:28: error: 'exported: true' has no effect in '_specialize' attribute
@_specialize(exported: true, where T == Int)
                           ^
<prj>/Pods/MobileMessaging/Classes/Vendor/CryptoSwift/Generics.swift:36:28: error: 'exported: true' has no effect in '_specialize' attribute
@_specialize(exported: true, where T == UInt)
                           ^
<prj>/MobileMessaging/Classes/Vendor/CryptoSwift/Generics.swift:37:28: error: 'exported: true' has no effect in '_specialize' attribute
@_specialize(exported: true, where T == UInt8)
                           ^
<prj>/Pods/MobileMessaging/Classes/Vendor/CryptoSwift/Generics.swift:38:28: error: 'exported: true' has no effect in '_specialize' attribute
@_specialize(exported: true, where T == UInt16)
                           ^
<prj>/Pods/MobileMessaging/Classes/Vendor/CryptoSwift/Generics.swift:39:28: error: 'exported: true' has no effect in '_specialize' attribute
@_specialize(exported: true, where T == UInt32)
                           ^
<prj>/Pods/MobileMessaging/Classes/Vendor/CryptoSwift/Generics.swift:40:28: error: 'exported: true' has no effect in '_specialize' attribute
@_specialize(exported: true, where T == UInt64)
```

Versions:
* MobileMessaging SDK version: 9.1.0
* Xcode version: 12.5
* Your application language: Swift 5.3
* Cocoapods version:  1.10.0

**Comment by kas-kad** [comment]
Hello @somenkovnikita 
Thanks for letting us know, we'll take care of it within next week and let you know of results.

**Comment by kas-kad** [comment]
Hello @somenkovnikita 
version 9.1.1 has been released containing the adjustments for Xcode 12.5 beta 3

**Comment by somenkovnikita** [comment]
> Hello @somenkovnikita 
> 
> version 9.1.1 has been released containing the adjustments for Xcode 12.5 beta 3

Wow, very fast! Many thanks!

**Comment by riskpp** [comment]
Closing the issue, because it was solved and we didn't get response.
